src/eko/harmonics/__init__.py

```diff
@@ -175,3 +175,44 @@ def s4x(n, sx, smx, is_singlet):
             sm31,
         ]
     )
+
+
+@nb.njit(cache=True)
+def compute_harmonics_cache(n, max_weight, is_singlet):
```

Comment:
we can drop "harmonics" from the name now (since we're inside the harmonics package (can be done here or in a separate PR)